src/main/java/com/example/application/ui/vertical/myProfile/MyProfileView.java

```diff
@@ -34,23 +34,33 @@
 @Route(value = "myProfile", layout = ContentHolder.class)
 @PageTitle("Mein Profil")
 
-
 public class MyProfileView extends Div {
     //These are only for demo, could be deleted once the data in the database is ready to be used
+    //private AddressesEntity addressesEntity = new AddressesEntity(1,"Beispielstraße", 123, 28359, "Bremen");
+
+    private MyProfileViewService myProfileViewService;
+    private PageEntity pageEntity;
+    private UsersEntity usersEntity;
+    private AddressesEntity addressesEntity;
+
 
-    private AddressesEntity addressesEntity = new AddressesEntity(1,"Beispielstraße", 123, 28359, "Bremen");
-    private UsersEntity usersEntity = new UsersEntity(1, "firstname", "surname", addressesEntity, "DE1273462537274", 'I', 5, 556434, "email", "wir sind toll" );
 
     /*
      * Constructor of the MyProfileVew class where the content is added to the view
      */
-    public MyProfileView() {
+    public MyProfileView(MyProfileViewService myProfileViewService) {
+        this.myProfileViewService = myProfileViewService;
+
         setId("myProfile-view");
         setClassName("pageContentPosition");
+
+        pageEntity = myProfileViewService.findPageById(2); //Only for demo, need to be setted by logged in user
+        usersEntity = pageEntity.getUser();
+        addressesEntity = usersEntity.getAddress();
+
```

Comment:
Verbindung von der AddressEntity zum User fehlt noch. Die AddressEntity, hat keinen User in der UsersListe. 
![image](https://user-images.githubusercontent.com/47325924/104212913-0859c480-5436-11eb-89a8-9575c109362e.png)

die fetch-Variable ist dafür da um einen Fehler zu beheben, mehr dazu hier: [proxy – no Session], ist nicht empfohlen, aber sollte momentan reichen (die Security Gruppe sollte sich um die Sessions kümmern) (https://www.baeldung.com/hibernate-initialize-proxy-exception)
![image](https://user-images.githubusercontent.com/47325924/104215904-b5820c00-5439-11eb-8a02-c7665e2b73a1.png)

in MyProfileView.java im Konstruktor
![image](https://user-images.githubusercontent.com/47325924/104216323-33461780-543a-11eb-9072-6ef02bcd5e2f.png)